BitwardenShared/UI/Auth/Login/TwoFactorAuth/TwoFactorAuthProcessor.swift

```diff
@@ -93,7 +93,7 @@ final class TwoFactorAuthProcessor: StateProcessor<TwoFactorAuthState, TwoFactor
             state.toast = newValue
         case let .verificationCodeChanged(newValue):
             state.verificationCode = newValue
-            state.continueEnabled = (newValue.count >= 6)
+            state.continueEnabled = newValue.count >= (state.deviceVerificationRequired ? 8 : 6)
```

Comment:
♻️ Maybe for another PR, but I'd extract `(state.deviceVerificationRequired ? 8 : 6)` to a private computed property that returns the minimum characters so to keep the logic simple here.